proto/storage/signature_integration.proto

```diff
@@ -19,3 +21,28 @@ message CosignPublicKeyVerification {
   }
   repeated PublicKey public_keys = 3;
 }
+
+// Holds all verification data for verifying certificates attached to cosign signatures.
+// In case both the certificate and chain are given, both will be used to verify the public key
+// as well as the chain.
```

Comment:
This reads oddly, if the chain is given, it will be used to verify the chain? Did you mean to remove line 27 here?